src/pages/employees/FormEmployee.tsx

```diff
@@ -44,13 +44,27 @@ const FormEmployee = (props: FormEmployeeProps) => {
     const [branchOfficeList, setBranchOfficeList] = useState<BranchOffice[]>([]);
     const [getEmployeeRoles] = useGetEmployeeRolesMutation();
     const [roleList, setRoleList] = useState<Role[]>([]);
+
+
     useEffect(() => {
-        handleGetEmployeeTypes();
-        handleGetBranchOffices();
-        handleGetRoles();
-        if (props.type == FormEmployeeType.UPDATE) {
-            handleSetupValues();
-        }
+        const fetchData = async () => {
+            try {
+                setTimeout(async () => {
+                    await handleGetEmployeeTypes();
+                    await handleGetBranchOffices();
+                    await handleGetRoles();
+                    if (props.type === FormEmployeeType.UPDATE) {
+                        handleSetupValues();
+                    }
+                    setIsLoading(false); 
+                }, 2000);
```

Comment:
can we use a constant for this value?

Reply:
Well, what exactly do you mean with this comment?